### PR TITLE
Develop to master - CKAN 2.10 prep

### PR DESCRIPTION
**Pull request by ThrawnCA**
- Update plugins and scenario tests to be compatible with CKAN 2.10
- Update DEV environment to CKAN 2.10 and Solr 8

### CODE REVIEW

File: dev-requirements-2.9.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+behaving==2.0.0
```
Comment (duttonw) [inline dev-requirements-2.9.txt:1]: is going to be awesome to shake off 2.9 holdbacks.

File: test/features/datasets.feature [hunk 2]
```diff
@@ -9,14 +9,12 @@ Feature: Dataset APIs
...
-        And I edit the "public-test-dataset" dataset
-        And I fill in "author_email" with "admin@example.com"
-        And I press the element with xpath "//form[@id='dataset-edit']//button[contains(@class, 'btn-primary')]"
+        And I create a dataset and resource with key-value parameters "notes=Testing activity stream" and "name=Test"
         And I press the element with xpath "//a[contains(@href, '/dataset/activity/') and contains(string(), 'Activity Stream')]"
         Then I should see "created the dataset"
         When I press "View this version"
         Then I should see "You're currently viewing an old version of this dataset."
-        When I go to dataset "public-test-dataset"
+        When I go to dataset "$last_generated_name"
```
Comment (duttonw) [inline test/features/datasets.feature:17]: awesome feature. :+1:

File: vars/shared-CKANTest.var.yml [hunk 3]
```diff
@@ -49,13 +49,13 @@ extensions:
...
       version: "v1.2.0"
 
-    CKANExtScheming: &CKANExtScheming
-      name: "ckanext-scheming-{{ Environment }}"
-      shortname: "ckanext-scheming"
-      description: "CKAN Extension for Scheming"
-      type: "git"
-      url: "https://github.com/ckan/ckanext-scheming.git"
-      version: "release-3.0.0"
+    #~ CKANExtScheming: &CKANExtScheming
```
Comment (duttonw) [inline vars/shared-CKANTest.var.yml:52]: need to reintroduce this to ckantest.

**Review by duttonw — CHANGES_REQUESTED** [review]
still have commented out yaml file for ckantest.

File: vars/shared-CKANTest.var.yml (hunk 3 above)
Comment (ThrawnCA) [reply vars/shared-CKANTest.var.yml:52]: Good spot, thanks.

https://github.com/qld-gov-au/ckan-qld-infrastructure/pull/384

File: vars/CKAN-Stack.var.yml [hunk 4]
```diff
@@ -45,7 +46,7 @@ common_stack: &common_stack
     EnableDataStore: "{{ enable_datastore | default('no') }}"
     SSMKey: "{{ SSMKey | default('') }}"
     DefaultEC2Key: "{{ lookup('aws_ssm', '/config/CKAN/ec2KeyPair', region=region) }}"
-    CookbookRevision: "{{ CookbookRevision | default('6.1.7') }}"
+    CookbookRevision: "{{ CookbookRevision | default('6.1.9') }}"
     LogBucketName: "{{ lookup('aws_ssm', '/config/CKAN/s3LogsBucket', region=region) }}"
     AttachmentsBucketName: "{{ lookup('aws_ssm', '/config/CKAN/' + Environment + '/app/' + service_name_lower + '/s3AttachmentBucket', region=region) }}" #/config/CKAN/PROD/app/opendata/s3AttachmentBucket
     SolrSource: "{{ solr7 }}"
```
Comment (duttonw) [inline vars/CKAN-Stack.var.yml:52]: are we swapping to solr8 only in dev or all?

**Review by duttonw — APPROVED** [review]

**Pull request merged**